src/types.h

```diff
@@ -46,14 +49,40 @@ typedef enum
     SP_SETREGID,
     SP_SETRESUID,
     SP_SETRESGID,
-} syscall_pattern_t;
+    SP_EXIT,
+    SP_EXITGROUP,
+    SP_UNSHARE,
+    SP_CLONE,
+    SP_CLONE3,
+    SP_FORK,
+    SP_VFORK,
+    SP_EXECVE,
+    SP_EXECVEAT,
+} syscall_pattern_type_t;
+
+typedef enum
+{
+    Unspec,
```

Comment:
can this be TE_UNSPEC to match convention

Reply:
750d697f2242e69233ef10fb19251be28b8c7bae